file.go

```diff
@@ -73,7 +75,7 @@ const (
 
 // Errors strings specific to parsing a Batch container
 var (
-	//msgFileCalculatedControlEquality = "calculated %v is out-of-balance with control %v"
+	// msgFileCalculatedControlEquality = "calculated %v is out-of-balance with control %v"
```

Comment:
What's this error message even for?

Reply:
That's a great question. I have no idea how long these have been commented out, but we can probably just delete these lines.